prom-otel-research/findings.md

```diff
@@ -0,0 +1,51 @@
+# Key Findings
+## Stakeholder Interviews
+We spoke to 6 stakeholders—3 from each project:
+
+**Prometheus stakeholders:**
+1. [Julius Volz](https://github.com/juliusv) - Prometheus Co-founder
+2. [Beorn Rabestein](https://github.com/beorn7) - Prometheus long-time maintainer
+3. [Richard Hartmann](https://github.com/RichiH) - OpenMetrics co-founder and Prometheus maintainer.
+
+**OpenTelemetry stakeholders:**
+1. [Juraci Paixão](https://github.com/jpkrohling) - OpenTelemetry Governance Committee member
+2. [Josh Suereth](https://github.com/jsuereth) - OpenTelemetry Technical Committee member
+3. [Austin Parker](https://github.com/austinlparker) - OpenTelemetry Governance Committee member and co-founder.
+
+Here are insights from our conversations with the stakeholders:
+- The Prometheus and OpenTelemetry communities hadn't always communicated well and that prevented them from collaborating early on.
+- Much of the interoperability issues that exist now stem from the different philosophical and technical foundations on which each of the projects is built. 
+> "If we think about exploratory situations or use cases, then we can justify a lot of the design decisions behind OpenTelemetry. And if we think about metrics and scaling, monitoring, for huge infra, then the design decisions for Prometheus are justified as well. So both have very good arguments." - Juraci Paixão
+
+> “I think one of the biggest (interoperability issue) is the difference between push and pull.” - Julius Volz
+- There’s a shared recognition of the importance of putting user needs first, even while maintaining some non-negotiables (like Prometheus keeping its pull model and not alienating existing users).
+
+> [!NOTE]
+> One key takeaway from the interviews was realizing that the current interoperability problems are not failures, but natural consequences of different communities solving different problems at different times. And it’s good to see both projects are working together now to make user experience better.
+
+## User Interviews Insights
+We interviewed 7 end users and they all shared incredibly helpful insights.
+- The most prominent pain point users shared was the **complexity of performing joins** with the current integration.
+- Regarding mental models, many users (both interviewees and survey respondents) don’t distinguish between resource attributes and Prometheus labels. They tend to think of them as the same thing.
+> "I would expect resource attributes as a rule to be treated exactly the same way as the attributes attached to the tracer, the metric... I wouldn't draw a boundary between them. I think Prometheus does at the moment." - Participant 1
+- As for workarounds, some users promote selected resource attributes to labels, others handle the conversion at the OpenTelemetry level to avoid dealing with it in Prometheus, and a few convert all attributes, though usually only when the number of attributes is small.
```

Comment:
```suggestion
- As for workarounds, some users promote selected resource attributes to labels, others handle the conversion at the OpenTelemetry-Collector level to avoid dealing with it in Prometheus, and a few convert all attributes, though usually only when the number of attributes is small.
```

If I remember correctly, they use some components from the collector to do these transformations